src/kategorier/kategorier.ts

```diff
@@ -10,10 +12,12 @@ export interface InngangKategori {
 export interface Kategori {
     temaKey: TemaKey;
     title: string;
-    anke: boolean;
+    allowsAnke: boolean;
     digitalKlage: boolean;
     digitalAnke: boolean;
     path: string;
+    kategoriMailKlageUrl: string;
```

Comment:
Litt usikker på hvorfor disse to har prefix `kategori`, mens de andre ikke har det.
Blir det uklart om vi korter de ned til `mailKlageUrl` og `mailAnkeUrl` her og andre steder?

Reply:
> Litt usikker på hvorfor disse to har prefix `kategori`, mens de andre ikke har det.
> Blir det uklart om vi korter de ned til `mailKlageUrl` og `mailAnkeUrl` her og andre steder?

Tror tankegangen min var dette er "mail-URLen til kategorien", men vi er jo allerede i kategori-kontekst, så tror det går fint å fjerne `kategori` som du sier.

Reply:
Ja, akkurat det jeg tenkte :)